charts/spire/charts/spire-agent/templates/daemonset.yaml

```diff
@@ -35,7 +35,7 @@ spec:
           # from https://github.com/vishnubob/wait-for-it
           image: {{ template "spire-agent.image" .Values.waitForIt }}
           imagePullPolicy: {{ .Values.waitForIt.image.pullPolicy }}
-          args: ["-t", "30", "-h", {{ .Values.server.host | quote }}, "-p", {{ .Values.server.port | quote }}]
+          args: ["-t", "30", "-h", "{{ .Release.Name }}-server", "-p", {{ .Values.server.port | quote }}]
```

Comment:
Shouldn't we use the 'spire-server-service' here too? (or the other way around, use this above.)

Reply:
Oh, I intented to remove that helper function.